crc.py

```diff
@@ -3,6 +3,7 @@
 # Copyright (c) 2018, Nicola Coretti
 # All rights reserved.
 import sys
+import io
```

Comment:
can be removed if we move the fix to the test itself.

Reply:
Right. Maybe merge your proposal then.

Reply:
I'll coming back to the change(s) in the evening, whatever you prefer.